linting/Cargo.toml

```
@@ -3,8 +3,21 @@ resolver = "2"
 members = [
     "mandatory",
     "extra",
+    "utils",
 ]
 
+[workspace.package]
+version = "5.0.0-rc"
```

you can imply the version from the global workspace
```suggestion
version.workspace = true
```

---

Actually, no. It is a definition of a separate workspace that is [excluded](https://github.com/paritytech/ink/blob/af90adceb05f45bc9fba755d57e62803f596b421/Cargo.toml#L21) from the main one, so we cannot access it here.